.github/workflows/terraform-apply.yml

```diff
@@ -0,0 +1,34 @@
+name: "Terraform apply"
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "terraform/**"
+
+env:
+  AWS_REGION: ca-central-1
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  TERRAFORM_VERSION: 0.13.6
+  TF_INPUT: false
+
+jobs:
+
+  terraform-apply:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@3d8debd658c92063839bc97da5c2427100420dec # v1.3.2
```

Comment:
could we use `cds-snc/terraform-tools-setup@v1` instead ?

Reply:
Good idea - I'll switch